src/redux/material/material.sagas.js

```diff
@@ -98,17 +142,30 @@ export function* handleMaterialDelete({ payload }) {
   }
 }
 
+export function* handleMaterialColorDelete({ payload }) {
+  try {
+    yield put(setMaterialLoading(true));
+    yield call(deleteMaterialColor, payload);
+    yield put(setMaterialLoading(false));
+    yield call(handleSuccessSnackbar, SUCCESS_DELETE_STATUS);
+    yield put(getMaterialColors(payload.id));
```

Comment:
I guess you can get your deleted color from your mutation request, then just filter your colors and update colors on redux state, it reduce useless request to the server.

Reply:
Fixed